src/retriever.jl

```diff
@@ -1,5 +1,17 @@
-module retriever
+module Retriever
 
-# package code goes here
+
+# Name all available datset scripts
+function datasets()
+    command = `retriever ls`
+    run(command)
```

Comment:
Yes, for reading the output we need to use something different. `readall` is deprecated in version 0.5, it is now replaced with `readstring` (https://docs.julialang.org/en/release-0.5/manual/running-external-programs/).